src/frontend/components/Felleskomponenter/Dropdowns/StyledDropdown.tsx

```diff
@@ -15,6 +15,7 @@ export interface StyledDropdownProps<ConstrainedString extends string> {
     label?: ReactNode;
     dynamisk?: boolean;
     bredde?: 'fullbredde' | 'xxl' | 'xl' | 'l' | 'm' | 's' | 'xs';
+    slektsforholdSøker?: boolean;
```

Comment:
Ah, ser nå at den typen jeg kommenterte opp ligger her. Ville lagt den i en egen type for slektforhold-dropdown i stedet for den generellee styleDropdown 😊 og kanskje renamet den til noe sånn gjelderSøker eller noe. Var litt vanskelig å skjønne ved første øyekast hva slektsforholdSøker betydde 🤔